src/main/java/com/softserveinc/dokazovi/dto/post/PostDTO.java

```diff
@@ -35,7 +35,7 @@ public class PostDTO {
 	private Timestamp createdAt;
 	@JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "dd.MM.yyyy")
 	private Timestamp modifiedAt;
-	@JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "dd.MM.yyyy")
+	@JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "MM.dd.yyyy HH:mm")
```

Comment:
Could you please clarify if your formatting was requested by front-end team?
Or is there another reason you applied different format than the bug-report holds?
`Expected
The date/time of the published card is displayed in <HH:MM, DD MMMM YYYY> format in the material after opening card` https://github.com/ita-social-projects/dokazovi-requirements/issues/563

Reply:
It also differs from 
`@JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "HH:mm dd.MM.yyyy")
	private Timestamp createdAt;`
It's better to keep the same formatting unless we have a substantial reason to keep them different.

Reply:
This formatting is a requirement of the front-end team. This change was agreed upon by the teams at Daily meeting.